pkgs/tools/security/pass-otp/default.nix

```diff
@@ -5,8 +5,8 @@ stdenv.mkDerivation {
   src = fetchFromGitHub {
     owner = "tadfisher";
     repo = "pass-otp";
-    rev = "f2feb3082324a91089782af9b7fbb71d34aa213d";
-    sha256 = "0iklvcfgw1320dggdr02lq3bc7xvnd2934l1w9kkjpbsfmhs955c";
+    rev = "9edb479405156f5712dd101661b86bd599904e8d";
```

Comment:
Please release tags here: `v1.1.0`: https://github.com/tadfisher/pass-otp/releases

Reply:
Version should be also part of the name.

Reply:
Done, thanks!